README.md

```diff
@@ -111,7 +111,7 @@ Real-world usage examples from code using ICU’s `UnicodeSet` which implements
 - All Greek code points that are either a letter, a mark (e.g. diacritic), or a decimal number:
 
     ```
-    [\p{Greek}&&[\p{Letter}\p{Mark}\p{Decimal_Number}]]
+    [\p{Script_Extensions=Greek}&&[\p{Letter}\p{Mark}\p{Decimal_Number}]]
```

Comment:
FYI  This was ported from an ICU UnicodeSet, where `\p{Greek}` means `\p{Script=Greek}` not Script_Extensions.
If you care to preserve that, you could switch to just `Script`; but it's probably not very important for this example.

[`[\p{Script_Extensions=Greek}-\p{Script=Greek}]`](https://util.unicode.org/UnicodeJsps/list-unicodeset.jsp?a=%5B%5Cp%7BScript_Extensions%3DGreek%7D-%5Cp%7BScript%3DGreek%7D%5D&abb=on&g=&i=)

Reply:
I know it maps to `Script` in other languages. The reason I went with `Script_Extensions` is that, IIUC, [it should generally be used instead of `Script`](https://www.unicode.org/reports/tr24/tr24-32.html#Script_Extensions) unless there is a good reason not to. I’d weakly prefer the patch as-is.

Reply:
(This might actually be a bug in the original code, since the pattern is explicitly looking for marks?)

Reply:
Script vs. Script_Extensions depends on what someone wants to do.
It's possible that the original code could be improved by switching to Script_Extensions. One would need to evaluate.
It's... not important for this example :-)